department-of-veterans-affairs/caseflow
# Queue | Colocated assign to other team

Resolves #6086

### Description
Adds a Send to Team option in Colocated actions dropdown for legacy tasks with `action` `:translation` or `:schedule_hearing`. For other tasks, [only Place hold and Send to Attorney options are available](https://dsva.slack.com/archives/C6E41RE92/p1535050388000100?thread_ts=1535046130.000100&cid=C6E41RE92). Adds SendToLocationModal which handles sending to team and assigning attorney.

### Acceptance Criteria 

### Testing Plan
1. Re-seed database (`rake db:seed`)
2. Go to `/queue` as `BVALSPORER`
3. Go to case details for task that's not Schedule Hearing or Translation type, confirm only `Place Hold` and `Send to Assigning Attorney` options present in actions dropdown
4. Go to case details for Schedule Hearing or Translation type task, confirm `Send to ${teamName} team` option available
5. Choose `Send to ${teamName} team`, click Send to Team in modal
6. Confirm task disappears from table view, task returned from server has `status: "completed"`

<details><summary>modal mockup</summary>
<img alt="screen_shot_2018-08-24_at_2 36 39_pm_720" src="https://user-images.githubusercontent.com/8533004/44662564-2eed4100-a9dc-11e8-8f03-c02700daa93b.png">
</details>

<details><summary>gif</summary>
<img src="https://user-images.githubusercontent.com/8533004/44662970-32cd9300-a9dd-11e8-805a-21e173e3c016.gif">
</details>
